src/test/java/org/folio/rest/impl/AccountsAPITest.java

```diff
@@ -149,20 +156,26 @@ public void eventIsPublishedWhenFeeFineIsClosedWithLoanAndNoRemainingAmount() {
     final JsonObject account = createAccountJsonObject(accountId)
       .put("loanId", loanId)
       .put("remaining", 90.00)
-      .put("status", createNamedObject("Open"));
+      .put("status", createNamedObject("Open"))
+      .put("dateClosed", null);
 
     accountsClient.create(account);
 
     final JsonObject updatedAccount = account.copy()
       .put("status", createNamedObject("Closed"))
       .put("paymentStatus", createNamedObject(PAID_FULLY.value()))
-      .put("remaining", 0.0);
+      .put("remaining", 0.0)
+      .put("dateClosed", DATE_CLOSED);
 
     accountsClient.update(accountId, updatedAccount);
 
     Response byId = accountsClient.getById(accountId);
     assertThat(byId, isPaidFully());
 
+    assertThat(accountsClient.getById(accountId).body().asString(), allOf(
```

Comment:
please remove allOf and toString in assertions, they are not needed

Reply:
I only use the assertion to check if the closing date changed after the update. I don't think it's necessary to remove them.